docs/ossec_alerts.md

```diff
@@ -40,7 +41,6 @@ These are the values you must specify in `prod-specific.yml`:
  * Email username to authenticate to the SMTP relay: `sasl_username`
  * Domain name of the email used to send OSSEC alerts: `sasl_domain`
  * Password of the email used to send OSSEC alerts: `sasl_password`
- * The fingerprint of your SMTP relay: `smtp_relay_fingerprint`
```

Comment:
Good point, the phrase "fingerprint" doesn't appear anywhere in the document now, so I'll work mention back in. However, the current implementation only supports `secure` and `fingerprint` types. Downgrading to `verify` would require editing the postfix template, rather than overriding a var. Trying not to bombard admins with config options, and stick to sane defaults.

If you think it's important for the security level to be easily overridden to any value on playbook runs, I can add support for that back in. If it's configurable, we should document it.